### DISCUSSION TITLE
Grafana su Home Assistant Core

### DISCUSSION THREAD

**Discussion by stepsolar** [indomus/forum#386] [HUB: Home Assistant]
Ciao a tutti, vi contatto per chiedervi se sarebbe possibile installare Grafana su Home Assistant Core. Se si come è possibile fare.. Grazie a tutti..

**Comment by KC-inDomus** [comment]
in che senso?

**Reply by stepsolar** [reply]
Non ho lo store ufficiale di home assistant perché il mio è versione core, ma vorrei installare Grafana come posso fare? (non sono un esperto ma qualcosa la capisco) Grazie

**Comment by gforment** [comment]
Ciao,
la configurazione ideale sulla base della mia esperienza è InfluxDB per registrare i dati e Grafana per la lettura/analisi. Io avevo installato entrambi su Docker (docker-compose). Se cerchi in rete Home Assistant Influx Grafana Docker trovi parecchia roba, su InDomus mi pare manchi una guida in merito.

**Reply by stepsolar** [reply]
Ed è anche possibile installare il tutto su Home Assistant Core?

**Reply by gforment** [reply]
non ha senso, cioè home assistant è una cosa, grafana un'altra... li installi separatamente e li integri (ossia Home Assistant scrive nel database di influxdb e grafa a legge da lì)

**Reply by stepsolar** [reply]
Ok, adesso sto capendo qualcosa in più. Cercherò allora qualcosa online e vi farò sapere. Se conoscete qualche guida utile sarebbe gradita. Grazie

**Reply by stepsolar** [reply]
Quindi non può essere installato usando questo https://github.com/hassio-addons/addon-grafana Giusto?

**Reply by gforment** [reply]
Non si possono postare link, una ricerca su Google propone articoli anche in italiano

**Reply by KC-inDomus** [reply]
> Non si possono postare link, una ricerca su Google propone articoli anche in italiano

no vabbè i link github sono autorizzati. cmq quello è un addon per home assistant os, nel caso specifico invece è consigliabile tirar su un container docker graphana a se stante

**Comment by Polvere79** [comment]
Ciao a tutti, stavo cercando anche io il modo di installare gli add on InfluxDB e Grafana presenti su Hassio-addons su Home Assistant core.
I docker posso essere installati anche su Core?

**Reply by snakuzzo** [reply]
La domanda è mal posta.
Se usi Core (che sia docker oppure no) tutti gli altri servizi che vuoi tirare su puoi tirarli su come ti pare (con docker oppure no)....è una scelta soggettiva.

**Reply by Polvere79** [reply]
E' mal posta perchè non sono esperto :)
Riproviamo così: qual è il metodo migliore per installare InfluxDB e Grafana su un Raspy 4 su cui gira HA Core (senza docker)?
Si possono comunque installare gli add-ons in qualche modo, anche senza avere Hassio e Supervisor?

**Reply by jovannig** [reply]
Ho provato ad installare influxdb e grafana su docker con docker-compose. 
Questo e' il mio docker-compose.yaml relativo ai due servizi:

`  
  grafana:
    image: rancher/grafana-grafana:latest
    container_name: grafana
    restart: always
    ports:
      - 3000:3000
    networks:
      - monitoring_network
    volumes:
      - grafana-data:/var/lib/grafana

  influxdb:
    image: monstrenyatko/rpi-influxdb:latest
    container_name: influxdb
    restart: always
    ports:
      - 8086:8086
    networks:
      - monitoring_network
    volumes:
      - influxdb-data:/var/lib/influxdb   

volumes:
  node-red-data:
  grafana-data:
  influxdb-data:

networks:
  node-red-net:
  monitoring_network:

`


Se valido il file con docker-compose config mi da' questo errore:
`
Pulling grafana (rancher/grafana-grafana:latest)...
ERROR: manifest for rancher/grafana-grafana:latest not found: manifest unknown: manifest unknown
`

Non ho trovato molto in rete sulla configurazione dei due servizi con docker compose. Potete aiutarmi?

**Reply by snakuzzo** [reply]
Per favore usa il markdown per il codice, altrimenti cancelleremo il post. Grazie

**Reply by jovannig** [reply]
Ho risolto, grazie. Avevo sbagliato immagine da scaricare. Mettendo grafana/grafana e' ok!
